scripts/validate

```diff
@@ -17,6 +17,9 @@ echo Running: golangci-lint
 golangci-lint run
 
 echo Tidying up modules
+cd pkg/apis/
```

Comment:
Make sure CI checks for this issue going forward

Reply:
Thanks for this!